sample-apps/python/django_frontend_service/frontend_service_app/views.py

```diff
@@ -50,9 +50,9 @@ def aws_sdk_call(request):
     bucket_name = "e2e-test-bucket-name"
 
     # Add an (pod) ID to bucketname to associate buckets to specific test runs
-    ip = request.GET.get('ip', None)
-    if ip is not None:
-        bucket_name += "-" + ip
+    bucket_id = request.GET.get('id', None)
+    if bucket_id is not None:
+        bucket_name += "-" + bucket_id
```

Comment:
Either use `id` for consistency with the Java app or use `testing_id` to make it clear where the value is supposed to come from. Also update the comment as per my other comment.